webdriver/tests/bidi/browsing_context/reload/reload.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+
+from pathlib import Path
+
+pytestmark = pytest.mark.asyncio
+
+PNG_BLACK_DOT = "/webdriver/tests/bidi/browsing_context/support/black_dot.png"
+
+
+async def test_return_value(bidi_session, inline, new_tab):
+    url = inline("<div>foo</div>")
+    await bidi_session.browsing_context.navigate(context=new_tab["context"],
+                                                 url=url)
+
+    result = await bidi_session.browsing_context.reload(
+        context=new_tab["context"])
+    assert result == {}
+
+
+@pytest.mark.parametrize(
+    "initial_url",
+    [
+        "about:blank",
+        "https://example.com/#foo",
+        "data:text/html,<p>foo</p>",
+        f'file://{Path(__file__).parent.parent.resolve() / "support/empty.html"}',
```

Comment:
Note that this will be an issue when running the tests on a device. But I'm fine with that given that other tests are affected as well.

Reply:
Ack. Then resolving this and not planning to touch this PR anymore. Thanks for your review!